src/goto-programs/scope_tree.h

```diff
@@ -90,6 +92,12 @@ class destructor_and_idt
 class scope_treet
 {
 public:
+  struct declaration_statet
+  {
+    goto_programt::targett instruction;
+    std::unordered_set<irep_idt, irep_id_hash> if_conditions_added;
```

Comment:
I have no idea what "if_conditions_added" means.

Reply:
I have now renamed and documented this in the two commits I just pushed. Please take another look to see if the meaning is clear to you now.